Modules/xASL_module_ASL.m

```diff
@@ -194,6 +193,16 @@
     fprintf('%.2f\n', round(x.Q.uniqueEchoTimes(end),4));
 end
 
+% If multi-TE data is detected, we switch on multi-TE quantification by default. Unless this has been deactived in dataPar.json
+if x.modules.asl.bMultiTE
+	if ~isfield(x.modules.asl, 'bMultiTEQuantification') || isempty(x.modules.asl.bMultiTEQuantification)
+		% By default, initialize as Multi-TE quantification as true
+		x.modules.asl.bMultiTEQuantification = true;
+	end
+else
+	x.modules.asl.bMultiTEQuantification = false;
```

Comment:
We should set this to false without multiple TEs as well right? Is just a default setting

Reply:
We already do that of course:
Without multiTEs, we set it to zero - always, even if set to true by the user.
In case of existing multiTEs, we set it to true if not initialized by the user. If initialized, we keep as set by the user.

Reply:
See my comments above, sounds like this parameter is moot

Reply:
I don't understand what parameter you talk about and what is the problem.